extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -4,8 +4,9 @@ require([
 	'wikia.mustache',
 	'communitypage.templates.mustache',
 	'wikia.nirvana',
-	'wikia.throbber'
-], function ($, uiFactory, mustache, templates, nirvana, throbber) {
+	'wikia.throbber',
+	'wikia.window'
+], function ($, uiFactory, mustache, templates, nirvana, throbber, window) {
```

Comment:
I am not sure if you show overwrite the `window` var with `wikia.window`. We usually use a different name like `w`.